File: home/sync.py
#!/usr/bin/env not-bash

home = Path("/home/matklad")
home_cfg = Path('/home/matklad/config/home')
for dir, _dirs, files in os.walk(home_cfg):
    for file in files:
        if file == "sync.py":
            continue

        abs_path = Path(dir) / file
        rel_path = abs_path.relative_to(home_cfg)
        dest = home / rel_path

        print(abs_path, '->', dest)

        try:
            dest.unlink()
        except:
            pass
        mkdir(dest.parent)
        os.symlink(abs_path, dest)

File: home/sync.jl
#!/usr/bin/env julia

config_home = "/home/matklad/config/home"
for (root, _dirs, files) in walkdir(config_home)
    for file in files
        file == "sync.jl" && continue

        abs_path = joinpath(root, file)
        rel_path = relpath(abs_path, config_home)
        dest = joinpath("/home/matklad/", rel_path)

        println("$abs_path -> $dest")

        rm(dest, force=true)
        mkpath(dirname(dest))
        symlink(abs_path, dest)
    end
end
